wolfcrypt/src/port/caam/wolfcaam_ecdsa.c

```diff
@@ -25,7 +25,7 @@
 
 #include <wolfssl/wolfcrypt/settings.h>
 
-#if defined(HAVE_ECC) && defined(WOLFSSL_QNX_CAAM)
+#if defined(WOLFSSL_QNX_CAAM) && defined(HAVE_ECC)
```

Comment:
Why change the order of the check here? It's functionally the same as before right?

Reply:
No different, just preference. Major Function then Minor Feature.